src/application/Explorer/Explorer.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useRef, useState, useEffect } from "react";
+import GraphiQL from "graphiql";
+import { Observable , useQuery, gql } from "@apollo/client";
+import { getIntrospectionQuery, buildClientSchema } from "graphql/utilities";
+import { parse } from "graphql/language/parser";
+import { print } from "graphql/language/printer";
+import GraphiQLExplorer from "graphiql-explorer";
+import { graphiQLQuery, ColorThemes } from '../index';
+import { sendGraphiQLRequest, receiveGraphiQLResponses, listenForResponse } from './graphiQLRelay';
+
+import "../../../node_modules/graphiql/graphiql.css";
+
+const GET_EXPLORER_DATA = gql`
+  query GetExplorerData {
+    colorTheme @client
+    graphiQLQuery @client
+  }
+`;
+
+enum FetchPolicies {
+  NoCache = 'no-cache',
+  CacheOnly = 'cache-only'
+}
+
+type FetchPolicy = FetchPolicies;
+
+export const Explorer = () => {
```

Comment:
Hmm, I left those components alone because the plan is a mass delete in the future. 

The new `Explorer` component lives outside `components` for a reason: the folder structure is **not** meant to  mirror the old one. The four major sections of the application are moved to the top-level:

index.js
/Explorer
/Queries
/Mutations
/Cache

And as I'm working on the navigation and layouts, those might live top-level too.

Reply:
Ahh, got it - cool with me, thanks!